boards/nxp/imx95_evk/imx95_evk_mimx9596_m7.yaml

```diff
@@ -18,4 +18,5 @@ supported:
   - i2c
   - pwm
   - spi
+  - gpio
```

Comment:
If no any GPIO dts node is enabled in board dts, it means that can't run any GPIO related test case on the board, so let's remove gpio from board yaml file which is used by twister to figure out which usecase can run on the board.

Reply:
Good point, removed from the yaml